doc/design/dist/submit-job.md

```diff
@@ -0,0 +1,70 @@
+
+# PaddlePaddle Client
+PaddlePaddle clinet is a command line tool, before startting a cluster train, you need to install it on your latop, you can sumit a cluster train job looks like:
+
+```bash
+paddle k8s job <job-name>
```

Comment:
Except `paddle k8s train`, also `paddle k8s [stop|status...]` command, and paddle client also support submit MPI cluster train job looks like: `paddle mpi train`, so maybe subcommand looks better?

`kubectl` style is good point, shall we support the two style, simple job configuration for command line parameters and complex job configuration use `yml` file.